Mathlib/MeasureTheory/Measure/OuterMeasure.lean

```diff
@@ -398,20 +415,22 @@ theorem supᵢ_apply {ι} (f : ι → OuterMeasure α) (s : Set α) : (⨆ i : 
 #align measure_theory.outer_measure.supr_apply MeasureTheory.OuterMeasure.supᵢ_apply
 
 @[norm_cast]
-theorem coe_supᵢ {ι} (f : ι → OuterMeasure α) : ⇑(⨆ i, f i) = ⨆ i, f i :=
-  funext fun s => by rw [supᵢ_apply, _root_.supr_apply]
+theorem coe_supᵢ {ι} (f : ι → OuterMeasure α) : ⇑(⨆ i, f i) = ⨆ i, ⇑(f i) :=
+  funext fun s => by simp
 #align measure_theory.outer_measure.coe_supr MeasureTheory.OuterMeasure.coe_supᵢ
 
 @[simp]
 theorem sup_apply (m₁ m₂ : OuterMeasure α) (s : Set α) : (m₁ ⊔ m₂) s = m₁ s ⊔ m₂ s := by
-  have := supᵢ_apply (fun b => cond b m₁ m₂) s <;> rwa [supᵢ_bool_eq, supᵢ_bool_eq] at this
+  have := supᵢ_apply (fun b => cond b m₁ m₂) s; rwa [supᵢ_bool_eq, supᵢ_bool_eq] at this
 #align measure_theory.outer_measure.sup_apply MeasureTheory.OuterMeasure.sup_apply
 
 theorem smul_supᵢ [SMul R ℝ≥0∞] [IsScalarTower R ℝ≥0∞ ℝ≥0∞] {ι} (f : ι → OuterMeasure α) (c : R) :
     (c • ⨆ i, f i) = ⨆ i, c • f i :=
   ext fun s => by
-    simp only [smul_apply, supᵢ_apply, ← smul_one_mul c (f _ _), ← smul_one_mul c (supᵢ _),
-      ENNReal.mul_supᵢ]
+  rw [smul_apply, ← smul_one_mul, supᵢ]
```

Comment:
Indentation again

Reply:
Could you please explain a bit more?
```
theorem smul_supᵢ [SMul R ℝ≥0∞] [IsScalarTower R ℝ≥0∞ ℝ≥0∞] {ι} (f : ι → OuterMeasure α) (c : R) :
    (c • ⨆ i, f i) = ⨆ i, c • f i :=
  ext fun s => by
  rw [smul_apply, ← smul_one_mul, supᵢ]
  simp only [supᵢ_apply, smul_apply, ← smul_one_mul c (f _ _)]
  rw [←supᵢ, ←ENNReal.mul_supᵢ]
  simp
#align measure_theory.outer_measure.smul_supr MeasureTheory.OuterMeasure.smul_supᵢ
```

`(c • ⨆ i, f i) = ⨆ i, c • f i :=` is part of the statement. So it has 2x indent.
`ext fun s => by` is the beginning of body. So it has 1x indent.
Am I misunderstanding the indentation rules?

Reply:
The tactics in the `by` block should be indented one more step.